FogCarport/src/main/java/presentation/orderCarport.java

```diff
@@ -0,0 +1,72 @@
+package presentation;
+
+import data.exceptions.LoginException;
+import data.models.CustomerModel;
+import data.models.OrderModel;
+import javax.servlet.http.HttpServletRequest;
+import logic.LogicFacade;
+import logic.LogicFacadeImpl;
+
+/**
+ * Call this from the .jsp when the advanced stuff is in place.
+ * @author 
+ */
+public class orderCarport extends Command
+{
+
+    public orderCarport()
+    {
+    }
+
+    @Override
+    String execute(HttpServletRequest request, LogicFacade logic) throws LoginException
+    {
+        LogicFacade db = new LogicFacadeImpl();
+        OrderModel order = new OrderModel();
+        CustomerModel customer = new CustomerModel();
+//        EmployeeModel employee = new EmployeeModel();
+        
+        // Customer Info
+        String name = request.getParameter("name");
+        int phonenumber = Integer.parseInt(request.getParameter("phonenumber"));
+        String email = request.getParameter("email");
+        String adress = request.getParameter("adress");
+        int zip = Integer.parseInt(request.getParameter("zip"));
+        // Set the Customer Info on Customer.
+        customer.setName(name);
+        customer.setEmail(email);
+        customer.setPhone(phonenumber);
+        customer.setAdress(adress);
+        customer.setZip(zip);
+        
+        // Carport Info
+        int length = Integer.parseInt(request.getParameter("length"));
+        int width = Integer.parseInt(request.getParameter("width"));
+        // Set the Carport Info on Order.
+        order.setLength(length);
+        order.setWidth(width);
+        

```

Comment:
Missing a few fields:  
```suggestion
//roof info
int roof_incline = request.getParameter("incline");
int roof_tiles_id = request.getParameter("roof_tiles_id");

order.setIncline(roof_incline);
order.setRoof_tiles_id(roof_tiles_id);
```